lib/components/sidebar/sidebar.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Drawer as AntDrawer } from 'antd';
+import Container from './container';
+import styles from './sidebar.css';
+
+const Sidebar = React.memo(({ visibleDrawer, cardsData, drawerProps }) => {
+  return (
+    <AntDrawer
+      {...{
+        className: styles.sidebarWrap,
+        visible: visibleDrawer,
+        closeIcon: null,
+        style: {
+          backgroundColor: 'var(--vds-color-slate-lightest)',
+        },
+        ...drawerProps,
```

Comment:
Instead of allowing all props to pass through, can we specify only the props that this component will allow? Since this is shared we want to limit how much customization can be done, and instead standardize this component for all applications.

Reply:
fixed